java-client/src/test/java/org/apache/pegasus/client/TestAdminClient.java

```diff
@@ -134,4 +137,35 @@ public void testDropApp() throws PException {
     pClient.close();
     Assert.fail("expected PException for openTable");
   }
+
+  @Test
+  public void testListApps() throws PException {
+    String appName = "testListApps" + System.currentTimeMillis();
+    List<app_info> appInfoList = new ArrayList<>();
+    toolsClient.listApps(true, appInfoList);
```

Comment:
Do you suppose there are all "available" tables in the cluster?

Reply:
It doesn't matter if there is no “available” table, a new “available” table will be created later.